Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :test do
```

Comment:
`require pry` could also be added to the spec_helper, giving us access to `binding.pry` without the require correct?